scripts/config.py

```diff
@@ -235,11 +235,15 @@ def full_adapter(name, active, section):
 
 # The baremetal configuration excludes options that require a library or
 # operating system feature that is typically not present on bare metal
-# systems. Features that are excluded from "full" won't be in "baremetal"
-# either (unless explicitly turned on in baremetal_adapter) so they don't
-# need to be repeated here.
+# systems. It also excludes debugging features that would typically be omitted
+# where code size is at a premium and that do not have a module of their own,
```

Comment:
"where code size is at a premium " or maybe "when code size is at a premium" ?

Reply:
I'm not sure what you're suggesting here, but this sentence was a bit hard to read so I've rewritten it.